packages/edit-site/src/components/sidebar-navigation-screen-details-footer/index.js

```diff
@@ -35,9 +36,9 @@ export default function SidebarNavigationScreenDetailsFooter( {
 	const lastRevisionId =
 		record?._links?.[ 'predecessor-version' ]?.[ 0 ]?.id ?? null;
 	const revisionsCount =
-		record?._links?.[ 'version-history' ]?.[ 0 ]?.count ?? 0;
-	// Enable the revisions link if there is a last revision and there are more than one revisions.
-	if ( lastRevisionId && revisionsCount > 1 ) {
+		recordCount || record?._links?.[ 'version-history' ]?.[ 0 ]?.count;
```

Comment:
Genuine question: is there an actual need to keep the 'or'? To my understanding, when the record count is falsy, the entire footer is not rendered because `shouldShowGlobalStylesFooter` is false in the parent component.

Reply:
The original purpose of this component was to sit at the bottom of template item views in which case the revisionsCount is obtained from the `_link` object in the record object. That's why I left it. Global styles revisions counts can be fetched from the `useGlobalStylesRevisions()` hook.